packages/experimental/TabList/src/TabIndicator/TabIndicator.types.ts

```diff
@@ -0,0 +1,26 @@
+import type * as React from 'react';
+
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import type { IForegroundColorTokens, LayoutTokens } from '@fluentui-react-native/tokens';
+
+export const tabIndicatorName = 'TabIndicator';
+
+export interface TabIndicatorCoreTokens extends IForegroundColorTokens {
+  inset?: number;
```

Comment:
How does this differ from Tab's inset token?

Reply:
They are pretty much the same. The value of Tab's `indicatorInset` token is passed directly here. 

While the TabIndicator component technically isn't needed, it does make the Tab styling code much easier to read and understand in my opinion.